### ISSUE TITLE
Simulator launching should be the first step when launch an app for simulator

### ISSUE THREAD

**Issue by BlueRiverInteractive** [robovm/robovm#796]
Currently the simulator will be launched as the last step which adds extra waiting time before an app can be run.
The simulator should be started before the app starts building. This way the simulator can take his time to startup while the app gets compiled and after the app is built, the simulator is probably ready to launch the app (or at least it doesn't take too long anymore).

**Comment by ntherning** [comment]
We need to patch `ios-sim` I believe to make it launch to simulator without any app specified?

**Comment by badlogic** [comment]
No, you can launch a Sim without an app. See command line help text of
ios-sim.
